src/FlashcardSystem.java

```diff
@@ -1,2 +1,122 @@
+import java.util.*;
+
 public class FlashcardSystem {
+    public DeckController deckController = new DeckController();
+    public SessionController sessionController = new SessionController();
+    public Scanner scanner = new Scanner(System.in);
+
+    public void displayMainMenu() {
+        System.out.println("(0) Logout (1) Create a Deck (2) Select deck to study (3) Edit deck");
+        String select = scanner.nextLine();
+        //TODO: check for invalid input
+        switch(select) {
+            case "0":
+                break;
+            case "1":
+                displayCreateDeckMenu();
+                break;
+            case "2":
+                displaySelectDeckToStudyMenu();
+                break;
+            case "3":
+                displayEditDeckMenu();
+                break;
+        }
```

Comment:
Currently, the UI just stops when we finish one of these cases. Could we use a do-while loop to make sure we always return to the main menu? Something like:
```suggestion
        String select;
        do {
            System.out.println("(0) Logout (1) Create a Deck (2) Select deck to study (3) Edit deck");
            select = scanner.nextLine();
            //TODO: check for invalid input
            switch(select) {
                case "0":
                    break;
                case "1":
                    displayCreateDeckMenu();
                    break;
                case "2":
                    displaySelectDeckToStudyMenu();
                    break;
                case "3":
                    displayEditDeckMenu();
                    break;
            }
        } while(!select.equals("0"));
```